### PR TITLE
[torch_xla2] Fix nn.functional.conv2d and conv3d

### PR DESCRIPTION
**Pull request by dvhg**
Replicate PyTorch behavior when using single number padding. Ref: https://pytorch.org/docs/stable/generated/torch.nn.functional.conv2d.html#torch.nn.functional.conv2d

### CODE REVIEW

**Review by ManfeiBai — APPROVED** [review]
Thanks, LGTM
